app/uk/gov/hmrc/gform/gform/handlers/FormValidator.scala

```diff
@@ -191,13 +191,15 @@ class FormValidator(implicit ec: ExecutionContext) {
           }
         )
       case FastForward.StopAt(to) :: xs =>
+        val maxTo = List(Some(to), nextFrom).max.get
```

Comment:
Please let's not use unsafe `.get` in production code

```suggestion
        val maxTo = nextFrom.fold(to)(n => List(to, n).max)
```

Reply:
committed your changes...